### ISSUE TITLE
Documentation bug: scss styles in Field Customizer tutorial are not applied

### ISSUE THREAD

**Issue by JLRishe** [SharePoint/sp-dev-docs#899]
#### Category
- [ ] Question
- [ ] Typo
- [X] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior
The tutorial on Field Customizers contains a step about replacing the contents of the .scss file and apply classes to the fields' html markup. The styles are intended to show the "progress bar" fields with a gray background.

#### Observed Behavior
Due to a bug in the documentation, this styling is not actually applied, and this is even reflected in the screenshot in the documentation. 

The reason for this is twofold:
- The hierarchy in the .scss styles places the `.cell` and `.full` classes inside a `.HelloWorld` class, but no `.HelloWorld` class is actually used anywhere in the markup, so the style rules don't actually match the HTML elements and they are not applied.
- The .scss uses string values for `background-color` and `width`, but using strings for these is invalid, so these rules are ignored even after the above is remedied.

The first issue can be remedied by either removing the enclosing `.HelloWorld` from the .scss file, or by wrapping the field HTML with a `<div class='${styles.HelloWorld}'>`.
The second issue can be remedied by removing the single quotes around the property values.

**Comment by VesaJuvonen** [comment]
thx Jimmy for reporting this. We'll get this processed and updated in the docs as soon as possible.

**Comment by VesaJuvonen** [comment]
Can you @JLRishe provide slightly more details on the issue? If you follow the tutorial steps as they are, there are no issues on getting styles to work in our customization. Following is after updating them properly. Are you saying that you can't make this work within your environment and if that's the case, can you share details around the used software. Thx.

![image](https://user-images.githubusercontent.com/7446437/31494251-2364ffb6-af5b-11e7-9b48-9b061672d32c.png)

**Comment by JLRishe** [comment]
@VesaJuvonen Thank you for your reply. 

To clarify what I mean: 

Looking at the .scss, it is clear to me that it is intended to do two things: 
- give the rendered "progress bar" a gray background
- make the full width of the "progress bar" 100 pixels

Even in the screenshot that you have just posted above, it's clear that neither of these is working: the `.full` element is 130 pixels wide instead of 100, and does not have a gray background. This is also true for the screenshots in the tutorial and the content in the YouTube video.

This is not an issue with my environment since the issue is exhibited in your own screenshots. I know why the issue is happening and how to fix it (I've detailed this in my initial post). I just think it was an oversight in the prepared materials.

I know this is a very minor detail, but I figured that since the tutorial includes the .scss file and has steps for using it, it would be good if it worked.

Below is what I believe the result is supposed to look like when the .scss styles are correctly applied:

![image](https://user-images.githubusercontent.com/6964029/31509065-1480254c-afbb-11e7-9eca-8b5e079d3182.png)
